### PR TITLE
add option to get the latest message using latest as request parameter

### PR DESCRIPTION
**Pull request by cpanato**
fixes: https://github.com/jhillyerd/inbucket/issues/50

### CODE REVIEW

File: smtpd/filestore.go [hunk 1]
```diff
@@ -181,6 +181,10 @@ func (mb *FileMailbox) GetMessage(id string) (Message, error) {
 		}
 	}
 
+	if id == "latest" {
+		return mb.messages[len(mb.messages)-1], nil
```
Comment (jhillyerd) [inline smtpd/filestore.go]: Won't this blow up if the mailbox is empty?  Can you add that to your unit test?

Comment (cpanato) [reply smtpd/filestore.go]: done, I overlooked that, I thought the entry point in the function checked that.

**Comment by jhillyerd** [comment]
Looks great, thank you

**Comment by cpanato** [comment]
@jhillyerd there any other steps to do? or you will merge to master?

**Comment by jhillyerd** [comment]
No more steps - is in master (rc2) now.

**Pull request merged**